Sources/SwiftQueue/SwiftQueue.swift

```diff
@@ -75,9 +75,9 @@ public protocol JobResult {
 
 /// Enum to define possible Job completion values
 public enum JobCompletion {
-
+    typealias T = Any
     /// Job completed successfully
-    case success
+    case success(T?)
```

Comment:
Why do we need this ?

Reply:
Sorry, it was my mistake, it is for a internal test, but I pushed to master.